diff --git a/node_modules/pdf-parse/lib/pdf-parse.js b/node_modules/pdf-parse/lib/pdf-parse.js
index 2320404..d82d49e 100644
--- a/node_modules/pdf-parse/lib/pdf-parse.js
+++ b/node_modules/pdf-parse/lib/pdf-parse.js
@@ -7,7 +7,7 @@
 'use strict';
 
 //see https://github.com/mozilla/pdf.js/blob/master/examples/node/pdf2svg.js
-const PDFJS = require('../pdf.js/v1.10.100/build/pdf.js');
+const PDFJS = require('pdfjs-dist/legacy/build/pdf.js');
 
 class PDFParser {
     static #max_pages_before_instance_is_destroyed = 10;
